books.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
+
```

Comment:
- [ ] Kindly make sure `Getters` and `Setters` for `@rentals` are present i.e you should update your `attr_accessor` attribute by also adding the `@rentals` variable

- [ ] Once you fix the above issue, make sure you also add a method for adding rentals where the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. For more details,  I suggest you read **[this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)**

Reply:
that method is being applied to the creation of the rental. which actually achieves the has-many relationship